src/test/resources/pattern.rle

```diff
@@ -0,0 +1,3 @@
+#C Generated by copy.sh/life
+x = 10, y = 9, rule = B3/S23
+2b2o$o2bo$2obob2o$bobo2bo$bo4bob2o$2b3obob2o$5bo$4bo$4b2o!
```

Comment:
The file format allows line breaks in the board data definition.
Try adding one, this is currently breaking your parser.
```suggestion
2b2o$o2bo$2obob2o$bobo2bo$
bo4bob2o$2b3obob2o$5bo$4bo$4b2o!
```